chain/jsonrpc/src/api/mod.rs

```diff
@@ -130,6 +130,7 @@ mod params {
 
         /// If value hasn’t been parsed yet, tries to deserialise it directly
         /// into `T` using given parse function.
+        #[allow(unused)]
```

Comment:
better remove it all together, no need to keep dead code, and we can always restore it if needed

Reply:
It looks more like a library to me and I'd never search through git history for needed methods, so I personally think it's better to leave it in case someone needs it.
But I don't have strong opinion here. Removing code is always a pleasure to me 😅

Reply:
rust will actually help you with it:
- if the method is public enough to be an API of the library it will not complain if unused
- if the method is not part of the API it will tell you that it is unused and ask to remove it
Let's do what rust is recommending :) If it warned you it's unused in the first place it means it's safe to remove it and we should.